source/tests/mock-kraken/main_ptref_test.cpp

```diff
@@ -139,6 +140,8 @@ struct data_set {
         b.data->pt_data->codes.add(b.lines["line:A"], "codeB", "Bise");
         b.data->pt_data->codes.add(b.lines["line:A"], "codeC", "C");
         b.data->pt_data->codes.add(b.sps.at("stop_area:stop1"), "code_uic", "bobette");
+        b.data->pt_data->codes.add(b.lines["line:A"]->network, "external_code", "A");
```

Comment:
minor : you can probably do
```suggestion
        b.data->pt_data->codes.add(b.networks["network:A"], "external_code", "A");
```

Reply:
Nope networks doesn't exist in builder